exec/vector/src/main/codegen/templates/NullableValueVectors.java

```diff
@@ -804,6 +1406,310 @@ public void fromNotNullable(${minor.class}Vector srce) {
       setValueCount(valueCount);
     }
   }
+
+  <#if type.major == "VarLen" && minor.class == "VarChar">
```

Comment:
Given that is (and much other code) is for a single class, does it really need to be generated via a template?